src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeSummaryRepositoryTest.java

```diff
@@ -163,4 +167,32 @@ void should_skip_bulkscan_container() {
                 )
             );
     }
+
+    @Test
+    void should_return_envelopes_received_and_status_summary_by_selected_date_created() {
+
+        Instant createdAt1 = instant("2021-03-17 11:32:26");
+        Instant createdAt2 = instant("2021-03-17 12:33:27");
+        Instant createdAt3 = instant("2021-03-17 12:35:27");
+        Instant createdAt4 = instant("2021-03-17 12:39:27");
+        Instant createdAt5 = instant("2021-03-17 12:49:27");
+        envelopeRepository.insert(new NewEnvelope(CONTAINER_1, FILE_1_1, createdAt1, null, Status.DISPATCHED));
+        envelopeRepository.insert(new NewEnvelope(CONTAINER_2, FILE_1_2, createdAt2, null, Status.REJECTED));
+        envelopeRepository.insert(new NewEnvelope(CONTAINER_1, FILE_2_1, createdAt3, null, Status.REJECTED));
+        envelopeRepository.insert(new NewEnvelope(CONTAINER_2, FILE_2_2, createdAt4, null, Status.CREATED));
+        envelopeRepository.insert(new NewEnvelope(CONTAINER_2, FILE_2_2, createdAt5, null, Status.CREATED));
```

Comment:
imagine you have "container3"  but not received any envelopes for this container. (so you should not insert any value to DB) but it should still appear in the result.

Reply:
ok done. It will appear in the result now